src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -102,7 +105,8 @@ public VendorListService(String cacheDir,
         cache = Objects.requireNonNull(createCache(fileSystem, cacheDir));
 
         fallbackVendorList = readFallbackVendorList(fallbackVendorListPath);
```

Comment:
It is not a part of this PR but let's make some refactoring: add checking for the path specified like `fallbackVendorListPath != null ? ... : null;`
So, then we can remove redundant condition in `readFallbackVendorList(..)` method.